pyfr/backends/openmp/kernels/errest.mako

```diff
@@ -3,37 +3,58 @@
 <%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
 
 void
-errest(int nrow, int nblocks, fpdtype_t *__restrict__ error,
+errest(int nrow, int nblocks, fpdtype_t *__restrict__ reduced,
```

Comment:
Can replace __restrict__ by restrict (the latter being standard C, the former an extension; unsure why we've historically not used it).